src/r7insight_js.js

```diff
@@ -410,20 +410,19 @@
         }
     };
 
-    function validate_region (region) {
+    function validate_region(region) {
         var allowed_regions = ['eu', 'us', 'ca', 'au', 'ap'];
         if (region) {
             if (allowed_regions.indexOf(region) > -1) {
                 return region;
             } else {
                 throw_error("Unrecognised region");
             }
-        } else {
-            throw_error("No region defined");
         }
+        throw_error("No region defined");
     }
 
-    function throw_error (message) {
+    function throw_error(message) {
```

Comment:
Is this really useful? 😂

Reply:
Yes, it helps you avoid writing a function for every single throw you want to do.
It looks stupid, I know, but it helps for nicer code.

Reply:
... Why dop you have to write a function for throws? :p All you're doing is just `throw "message"`

Reply:
Ok so, since I separated the `validate_region` into a function then the `throw` lives inside a function, which is great.
Initially, it wasn't in a function and it gave out.

TLDR;
I removed the `throw_error` function cause it was no longer needed.